databricks/README.md

```diff
@@ -0,0 +1,160 @@
+## Spice on Databricks
+
+Spice can  read data straight from your Databricks instance. This guide will create an app, configure Databricks, load and query a dataset. It assumes:
+- The Databricks instance is running against AWS S3 storage in `us-east-1`.
+- Basic AWS authentication is configured (with environment variable credentials `AWS_ACCESS_KEY_ID` & `AWS_SECRET_ACCESS_KEY`).
+- A Databricks personal access token is available (as the environment variable `DATABRICKS_TOKEN`).
+- A table already exists in Databricks, called `spice_data.public.awesome_table`.
+
+1. First, install Spice (see the [Getting Started](https://docs.spiceai.org/getting-started) documentation).
```

Comment:
Move this "first, install spice" comment into the assumes section since you're not giving explicit instructions.  

it assumes:
- Spice is installed (see the [Getting Started](https://docs.spiceai.org/getting-started) documentation).